ERROR
2022-11-11 16:08:45,714
Unhandled exception in listener callback
Traceback (most recent call last):
  File "c:\Users\alexa\Desktop\pro_110822\pro_110822_venv\lib\site-packages\pynput\_util\__init__.py", line 228, in inner
    return f(self, *args, **kwargs)
  File "c:\Users\alexa\Desktop\pro_110822\pro_110822_venv\lib\site-packages\pynput\keyboard\_win32.py", line 287, in _process
    self.on_press(key)
  File "c:\Users\alexa\Desktop\pro_110822\pro_110822_venv\lib\site-packages\pynput\_util\__init__.py", line 144, in inner
    if f(*args) is False:
  File "c:\Users\alexa\Desktop\pro_110822\pro_110822_venv\lib\site-packages\pynput\keyboard\__init__.py", line 223, in _on_press
    hotkey.press(self.canonical(key))
  File "c:\Users\alexa\Desktop\pro_110822\pro_110822_venv\lib\site-packages\pynput\keyboard\__init__.py", line 182, in press
    self._on_activate()
  File "<string>", line 1, in <lambda>
  File "c:\Users\alexa\Desktop\pro_110822\pro_110822\mainwindow.py", line 116, in _on_shortcut_activate
    print(f"passed {self.sendmouseMovmentSockets[-1]['socket']}")
IndexError: list index out of range
